asm/vm/asm_vm

```diff
@@ -207,53 +224,57 @@ All options can also be passed via environment variables by using the ALL_CAPS
 name. Options specified via flags take precedence over environment variables.
 
 POSITIONAL ARGUMENTS:
-  NAME                                              Name of the instance
-                                                    template to create.
+  NAME                                                Name of the instance
+                                                      template to create.
 
 OPTIONS:
-  -l|--cluster_location         <LOCATION>          The GCP location of the
-                                                    target cluster.
-  -n|--cluster_name             <NAME>              The name of the target
-                                                    cluster.
-  -p|--project_id               <ID>                The GCP project ID.
-  -w|--workload_name            <WORKLOAD_NAME>     The name of the workload
-                                                    running on the GCE
-                                                    instance(s).
-  -e|--workload_labels          <WORKLOAD_LABELS>   (optional) The
-                                                    labels of the workload,
-                                                    comma separated.
-  -i|--source_instance_template <TEMPLATE>          (optional) The name of the
-                                                    source GCE instance
-                                                    template.
-  -s|--service_account          <SERVICE_ACCOUNT>   (optional) The name of a
-                                                    service account used to run
-                                                    the script.
-  -k|--key_file                 <FILE PATH>         (optional) The key file for
-                                                    a service account. Required
-                                                    if --service_account is
-                                                    passed.
+  -l|--cluster_location         <LOCATION>            The GCP location of the
+                                                      target cluster.
+  -n|--cluster_name             <NAME>                The name of the target
+                                                      cluster.
+  -p|--project_id               <ID>                  The GCP project ID.
+  -w|--workload_name            <WORKLOAD_NAME>       The name of the
+                                                      WorkloadGroup for GCE VM
+                                                      workloads.
+  --workload_namespace          <WORKLOAD_NAMESPACE>  (optional)The namespace of
```

Comment:
nit - space after '(optional)' and possibly ident to match other flags

Reply:
Done.